src/environment.development.json

```diff
@@ -2,6 +2,10 @@
   "release_id": "development",
   "messages": [],
   "configs": {
+    "google_analytics": [
+      "UA-73668038-11",
+      "G-0D68920QV9"
+    ],
```

Comment:
I was going to say to put a comment here so we remember to remove it later but it's a json file :/